modules/data-entry/src/main/frontend/src/questionnaire/FormEntry.jsx

```diff
@@ -70,6 +70,12 @@ let displayQuestion = (questionDefinition, path, existingAnswer, key, classes, o
   const existingQuestionAnswer = existingAnswer && Object.entries(existingAnswer)
     .find(([key, value]) => value["sling:resourceSuperType"] == "lfs/Answer"
       && value["question"]["jcr:uuid"] === questionDefinition["jcr:uuid"]);
+
+  // Do not show anything if in view mode and no value is recorded yet
+  if (!existingQuestionAnswer?.[1]["displayedValue"] && !isEdit && questionDefinition.dataType != "somaticVariantsFile") {
```

Comment:
Bug: a question with notes, but not an actual answer will not be displayed. For example, create a Radiotherapy form, don't choose any option for the "Radiation therapy" question, but enter notes for it. In view mode, nothing will be displayed, unless an answer is chosen.
```suggestion
  if (!(existingQuestionAnswer?.[1]["displayedValue"] || existingQuestionAnswer?.[1]["note"]) && !isEdit && questionDefinition.dataType != "somaticVariantsFile") {
```